CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
 ## New Functionality
 
 - Support custom axios options for all request builders.
+- Support disabling csrf token request as an option for all request builders.
```

Comment:
BTW: here and above the "tag is missing" [odata-common] mostly, e. g.

Reply:
I did not put any tags because these are something like [common] for both odata and openapi. Maybe I put [core] for both?